Projects/App/Sources/UgglanStore.swift

```diff
@@ -47,6 +47,9 @@ public final class UgglanStore: StateStore<UgglanState, UgglanAction> {
                     query: GraphQL.FeaturesQuery(),
                     cachePolicy: .fetchIgnoringCacheData
                 )
+                .mapError({ error in
+                    .init(member: .init(features: []))
```

Comment:
This caused an endless spinenr when this query was failing, so I guess its good to fallback to empty.